hoomd/logging.py

```diff
@@ -625,25 +628,49 @@ class they come from. For instance, the `hoomd.md.pair.LJ` class has a
         occur for user specified names that are reused.
 
     Args:
-        categories (`list` [`str`], optional): A list of string categories
-            (list of categories can be found in `LoggerCategories`). These are
-            the only types of loggable quantities that can be logged by this
-            logger. Defaults to allowing every type.
+        categories (`list` [`str` ], `LoggerCategories`, optional): Either a
+            list of string categories (list of categories can be found in
+            `LoggerCategories`) or a `LoggerCategories` instance with the
+            desired flags set. These are the only types of loggable quantities
+            that can be logged by this logger. Defaults to allowing every type
+            ``LoggerCategories.ALL``.
         only_default (`bool`, optional): Whether to log only quantities that are
             logged by default. Defaults to ``True``. Non-default quantities
             are typically measures of operation performance rather than
             information about simulation state.
 
-    .. rubric:: Example:
+    .. rubric:: Examples
+
+    There are various ways to create a logger with different available
+    loggables. To allow all loggable categories just create a logger.
```

Comment:
```suggestion
    loggables. Create a `Logger` with no options to allow all categories.
```
"just" belittles the reader.